codec/codec-fury/src/main/resources/system.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
test config should be put into test resources.

Reply:
IF multiple jar both provides `system.json`,  which one will be parsed? We may need both of them being parsed and merged into one config. ditto for user.json

Reply:
The name of system.json/users.json files may need change, seems a little  vague to me.

Reply:
Yes, there is definitely something to be changed here. But in the end what form to get the configuration file, they actually did not decide.